test/services/assets/handlers/sas.spec.js

```diff
@@ -486,65 +486,67 @@ describe('services', () => {
         expect(libraries[0]).toMatchObject({
           id: 'ods excel - ""',
           type: 'data',
-          path: '""'
+          path: '""',
         });
         libraries = new SASHandler().getOutputs('test.uri', 'ods csvall file="test.csv";');
         expect(libraries.length).toEqual(1);
         expect(libraries[0]).toMatchObject({
           id: 'ods csvall - "test.csv"',
           type: 'data',
-          path: '"test.csv"'
+          path: '"test.csv"',
         });
         libraries = new SASHandler().getOutputs('test.uri', "ods html5 body='test.html';");
         expect(libraries.length).toEqual(1);
         expect(libraries[0]).toMatchObject({
           id: "ods html5 - 'test.html'",
           type: 'data',
-          path: "'test.html'"
+          path: "'test.html'",
         });
         libraries = new SASHandler().getOutputs('test.uri', 'ODS Html File="test.html";');
         expect(libraries.length).toEqual(1);
         expect(libraries[0]).toMatchObject({
           id: 'ODS Html - "test.html"',
           type: 'data',
-          path: '"test.html"'
+          path: '"test.html"',
         });
         libraries = new SASHandler().getOutputs('test.uri', 'ODS CSVALL\r\n  FILE="test.csv";');
         expect(libraries.length).toEqual(1);
         expect(libraries[0]).toMatchObject({
           id: 'ODS CSVALL - "test.csv"',
           type: 'data',
-          path: '"test.csv"'
+          path: '"test.csv"',
         });
       });
       it('should ignore save locations for data in invalid statements', () => {
         // Missing semicolon
         expect(
-          new SASHandler().getOutputs('test.uri', 'ods csvall file="test.csv"').length
+          new SASHandler().getOutputs('test.uri', 'ods csvall file="test.csv"').length,
         ).toEqual(0);
         // Missing quotes
         expect(new SASHandler().getOutputs('test.uri', 'ods csvall file=test.csv').length).toEqual(
-          0
+          0,
         );
         // Invalid command
         expect(new SASHandler().getOutputs('test.uri', 'ods csval file="test.csv"').length).toEqual(
-          0
+          0,
         );
       });
       it('should ignore commented data save locations', () => {
         expect(
-          new SASHandler().getOutputs('test.uri', '* ods csvall file="test.pdf";').length
+          new SASHandler().getOutputs('test.uri', '* ods csvall file="test.pdf";').length,
         ).toEqual(0);
         expect(
-          new SASHandler().getOutputs('test.uri', ' *ods excel file="test.xlsx";').length
+          new SASHandler().getOutputs('test.uri', ' *ods excel file="test.xlsx";').length,
         ).toEqual(0);
         // eslint-disable-next-line prettier/prettier
-        expect(new SASHandler().getOutputs('test.uri', '  *   ods excel file="test.xlsx";  ').length).toEqual(0);
+        expect(
```

Comment:
I'm okay to accept the linter suggestion here, but we should remove the eslint-disable line on 541